lib/client/mfa.go

```diff
@@ -76,6 +91,9 @@ func PromptMFAChallenge(
 		hasWebauthn = false
 	}
 
+	// Prompt only for the strongest auth method available?
+	hasTOTP = !(opts.UseStrongestAuth && hasWebauthn) && hasTOTP
```

Comment:
nit. I stared at this line for quite some time try to understand it. is this the same as
```suggestion
	if opts.UseStrongestAuth && hasWebauthn {
		hasTOTP = false
	}
```
up to you if you prefer to keep the one liner.

Reply:
I agree, this seems better. That's what I started with, but then I felt the urge to "simplify".